app/main.py

```diff
@@ -4,6 +4,10 @@ def copy_file(command: str) -> None:
     if old_file == file_copy and command == "cp":
```

Comment:
I think we can add block "with open" into condition if old_file == file_copy and command == "cp": and dont use if command != "cp":`. If command is not correct, we just we don't get into the block with open